www/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Landing page for the Slinkity build plugin. To eleventy and beyond!",
   "main": ".eleventy.js",
   "scripts": {
-    "start": "slinkity --serve --incremental",
     "build": "slinkity",
+    "serve": "slinkity --serve --incremental",
```

Comment:
Update the `www/README.md` to reflect this change!

Reply:
OK. i'll update the docs and It may also be that `dev` is a better name for the script than `serve`

Reply:
Yeah I think `dev` will be more in-line with other repos 👍